apps/src/studio/levels.js

```diff
@@ -753,11 +753,7 @@ levels.playlab_6 = utils.extend(levels.move_penguin, {
 levels.iceage_6 = utils.extend(levels.playlab_6, {
   background: 'tile',
   firstSpriteIndex: 3, // diego
-  goalOverride: {
-    goal: 'ia_projectile_1',
-    success: 'ia_projectile_1',
-    imageWidth: 800
-  }
+  goalOverride: {}
```

Comment:
When I remove this, it will inherit the goal override from original playlab, which won't work because the projectile names aren't the same.